docs/releasing.md

```diff
@@ -0,0 +1,13 @@
+# Releasing uPortal-home
+
+## Apache release rules
+
+This project has [adopted Apache rules][] with necessary or pragmatic local adaptations.
```

Comment:
Indeed. Unwritten "necessary or pragmatic local adaptations" to rules is obnoxious and totally unacceptable. It's just that it's less obnoxious than any of the immediately feasible alternatives. I don't know how to write down all the adaptations at this point, because I don't know what all they are yet.

I agree about the value in paying down process documentation debt. Let's continue to identify and chase down actionable steps on this through the incubation process. [Governance is an item on the incubation checklist](https://github.com/UW-Madison-DoIT/angularjs-portal/blob/master/docs/apereo-incubation.md#43-governance), e.g. I'd readily agree that where we're at on figuring this out doesn't meet the Incubation exit threshold.